src/app/clusters/test-cluster-server/test-cluster-server.cpp

```diff
@@ -923,7 +923,7 @@ bool emberAfTestClusterClusterTestListInt8UReverseRequestCallback(
         Commands::TestListInt8UReverseResponse::Type responseData;
         if (count == 0)
         {
-            SuccessOrExit(commandObj->AddResponseData(commandPath, responseData));
+            SuccessOrExit(commandObj->AddResponse(commandPath, responseData));
```

Comment:
The exit block here calls emberAfSendImmediateDefaultResponse.  Followup to fix the logic here is fine.

Reply:
The exit block should not be changed, but SuccessOrExit is not necessary here.